docs/functions/index.mdx

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 1
+sidebar_label: Momento Functions
+title: Momento Functions
+description: Process real-time data at massive scale with high-performance Momento Functions.
+hide_title: true
+slug: /functions
+keywords:
+  - momento
+  - functions
+  - wasm
+  - real-time processing
+  - event-driven
+  - serverless
+---
+
+# Momento Functions
+
+*Momento Functions* are a high-speed compute layer designed for processing massive amounts of real-time data. Whether you're aggregating IoT sensor data, handling financial transactions, or enriching event streams with AI, *Momento Functions* allow you to process, transform, and act on data instantly—without managing infrastructure.
+
+Built for extreme scale and performance, functions offer low-latency execution and shared global memory across all running function instances. Unlike general-purpose serverless compute, they do not execute in isolation, allowing all instances of a function to share state for efficient data processing.
+
+```mermaid
+mindmap
+  root((Momento Functions))
+    (Fast Execution)
+      (5ms SLA for ultra-fast processing)
```

Comment:
I don't think we're offering an SLA. Some functions will take much longer to execute than 5ms, and it's largely outside of Momento's control how long they take. We can't make Bedrock go any faster than it does, and we can't exceed the universe's rate of information propagation (C).
Maybe we can say something about sub-millisecond cold starts, if you want to add a latency number? I think people typically associate that number with the right technical detail.